helm/featurehub/values.yaml

```diff
@@ -69,31 +365,128 @@ dacha:
+  # dacha deployment. Create the secret in advance, then reference it here.
+  # As of 1.5.0 all FeatureHub controller properties are available as environment variables using the same case
+  # Entries of the secret specified here are the same as would be specified in /etc/app-config/secrets.properties
+  envFromSecret: ""
+
+  # -- Prometheus configuration
+  prometheus:
+    # -- Whether to enable or disable prometheus metrics endpoints, and serviceMonitor
+    # If enabled, metrics are exposed on port 8701, on /metrics endpoint
+    enabled: false
+    # -- Labels for the Prometheus Operator to handle the serviceMonitor
+    labels: {}
+
+  ingress:
+    enabled: false
+    className: ""
+    annotations: {}
+      # kubernetes.io/ingress.class: nginx
+      # kubernetes.io/tls-acme: "true"
+    hosts: []
+      #  - host: chart-example.local
+      #    paths:
+      #      - path: /
+      #        pathType: ImplementationSpecific
+    tls: []
+      #  - secretName: chart-example-tls
+      #    hosts:
+      #      - chart-example.local
+
+  imagePullSecrets: []
+
+  podAnnotations: {}
+
+  podSecurityContext:
+    fsGroup: 999
+
+  securityContext:
+    runAsNonRoot: true
+    runAsUser: 999
+      # capabilities:
+      #   drop:
+      #   - ALL
+      # readOnlyRootFilesystem: true
+
+  # -- List of extra containers to add to Dacha Pod
+  extraContainers: []
+    #- name: do-something
+    #  image: busybox
+    #  command: ['do', 'something']
+
+  # -- List of extra mounts to add to Dacha Deployment
+  extraVolumeMounts: []
+    #- mountPath: /tmp/foobar
+    #  name: foobar
+
+  # -- List of extra volumes to add to Dacha Deployment
+  extraVolumes: []
+    #- name: foobar
+    #  emptyDir: {}
+
+# ------------------------------------------------------------------------------- #
+# NATS Helm Chart from Bitnami
+# Documentation: https://github.com/nats-io/k8s/tree/main/helm/charts/nats
+# ------------------------------------------------------------------------------- #
 nats:
   enabled: true
-  urls: nats://featurehub-nats:4222
   cluster:
     enabled: true
     name: featurehub
     replicas: 3
-  topologyKeys:
-    - "kubernetes.io/hostname"
-    - "topology.kubernetes.io/zone"
-    - "topology.kubernetes.io/region"
+  topologyKeys: []
+    #- "kubernetes.io/hostname"
+    #- "topology.kubernetes.io/zone"
+    #- "topology.kubernetes.io/region"
 
-# this is not intended for any kind of production use, please don't use it as such.
-# For more information on values which can be set see: https://github.com/bitnami/charts/tree/master/bitnami/postgresql
+# ------------------------------------------------------------------------------- #
+# PostgreSQL Helm Chart from Bitnami:
+# Documentation: https://github.com/bitnami/charts/tree/master/bitnami/postgresql
+# ------------------------------------------------------------------------------- #
 postgresql:
-  enabled: true
+  enabled: false
```

Comment:
this needs to be enabled: true by default so someone just bringing it up can use it. I know no-one will in real life, but it just means its not a working example otherwise...

Reply:
That's a quick one, I'll do it now. Thanks for your comments!

Reply:
Now it is set to `enabled: true` by default, I've added also a comment to indicate that it is not a hard requirement having it set to true.

Thanks!